### PR TITLE
React toggle switch component

### PR DESCRIPTION
**Pull request by chris-ys-kim**

### CODE REVIEW

File: react-hot-button-component/src/hot-button.jsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  }
+
+  numberClick() {
+    this.setState({ count: this.state.count + 1 });
+  }
+
+  render() {
+    let className = 'white';
+    const count = this.state.count;
+    if (count >= 3 && count < 6) {
```
Comment (Zhiwutian) [inline react-hot-button-component/src/hot-button.jsx:17]: Make sure to move this logic to a separate method as in the previous challenge to clean up the render method.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
Nice work sir! One update, and make sure to include screenshots when you update the pull request!

**Comment by chris-ys-kim** [comment]
Screen Shot 2021-07-09 at 5 20 21 PM
Screen Shot 2021-07-09 at 5 20 17 PM

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**